src/player.js

```diff
@@ -112,6 +112,17 @@ export default class Player extends FakeEventTarget {
    * @static
    */
   static _engines: Array<typeof IEngine> = [Html5];
+
+  /**
+   * Runs the engines tests for autoplay capabilities.
+   * @static
+   * @public
+   * @returns {void}
+   */
+  static testAutoPlayCapabilities(): void {
+    Player._engines.forEach(Engine => Engine.canAutoPlay());
```

Comment:
for future proof I would expose a capabilities object that can be quarried from player.
It can expose all player capabilities and by nature would be a promise.all that awaits all capabilities metrics to be finished.
Exposing each capability on it's own will result in API bloating

Reply:
of course, while implemented this I cannot think of other capability for generic impl :)

Reply:
ok, but we can already generalize the API to be more general like getCapabilities ()
which will return a promise that resolves all capabilities API calls.
This can be exposed in the Playkit API where you currently expose only autoplay explicitly